lib/mix/tasks/install_schemas.ex

```diff
@@ -0,0 +1,126 @@
+defmodule Mix.Tasks.Lightning.InstallSchemas do
+  @shortdoc "Install the credential json schemas"
+
+  @moduledoc """
+  Install the credential json schemas
+  Use --exclude language-package1, language-package2 to exclude specific packages
+  """
+
+  use Mix.Task
+  use HTTPoison.Base
+
+  @schemas_path "priv/schemas/"
+  @default_excluded_adaptors [
+    "language-common",
+    "language-devtools"
+    # "language-divoc",
+    # "language-bigquery",
+    # "language-twilio"
+  ]
+
+  @spec run(any) :: any
+  def run(args) do
+    HTTPoison.start()
+
+    init_schema_dir()
+
+    args
+    |> parse_excluded()
+    |> fetch_schemas()
+  end
+
+  def parse_excluded(args) do
+    args
+    |> case do
+      ["--exclude" | adaptor_names] when adaptor_names != [] ->
+        (adaptor_names ++ @default_excluded_adaptors) |> Enum.uniq()
+
+      _ ->
+        @default_excluded_adaptors
+    end
+  end
+
+  defp init_schema_dir() do
+    File.rm_rf(@schemas_path)
+
+    File.mkdir_p(@schemas_path)
+    |> case do
+      {:error, reason} ->
+        raise "Couldn't create the schemas directory: #{@schemas_path}, got :#{reason}."
+
+      _ ->
+        nil
+    end
+  end
+
+  def write_schema(package_name, data) when is_binary(package_name) do
+    file =
+      File.open!(
+        @schemas_path <>
+          String.replace(package_name, "@openfn/language-", "") <> ".json",
+        [:write]
+      )
+
+    IO.binwrite(file, data)
+    File.close(file)
+  end
+
+  def persist_schema(package_name) do
+    get(
+      "https://cdn.jsdelivr.net/npm/#{package_name}/configuration-schema.json",
+      [],
+      hackney: [pool: :default],
+      recv_timeout: 15_000
+    )
+    |> case do
+      {:error, _} ->
+        raise "Unable to access #{package_name}"
+
+      {:ok, %HTTPoison.Response{status_code: 200, body: body}} ->
+        write_schema(package_name, body)
+
+      {:ok, %HTTPoison.Response{status_code: status_code}} ->
+        raise "Unable to fetch #{package_name} configuration schema. status=#{status_code}"
```

Comment:
I think we need to adjust this behaviour here, when a language pack doesn't have a configuration schema the whole stream stops - resulting in an inconsistent set of results. For example `language-divoc` doesn't have a schema yet.

For now I think logging a warning instead of throwing is about as strict as we can be.